src/client/src/client_main.cpp

```diff
@@ -0,0 +1,66 @@
+//
+// Created by kevin on 6/25/22.
+//
+#include <ros/ros.h>
+#include <client/current_time.h>
+#include <client/login.h>
+
+#include <ctime>
+#include <cstring>
+#include <iostream>
+
+ros::ServiceClient service_client;
+client::login login_service_message;
+
+int main(int argc, char **argv) {
+    std::string main_name;
+    bool logged = 0;
+    std::cin>>main_name;
+    ros::init(argc, argv, main_name);
+    ros::NodeHandle client_node_handle;
+
+    client::current_time publishing_msg;
+    ros::Publisher client_publisher =
+            client_node_handle.advertise<client::current_time>(ros::this_node::getName(), 1);
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]